Doc/library/fileinput.rst

```diff
@@ -175,6 +175,8 @@ available for subclassing as well:
    .. versionchanged:: 3.8
       The keyword parameter *mode* and *openhook* are now keyword-only.
 
+   .. versionchanged:: 3.9
```

Comment:
Remove the `deprecated` directive above.

Reply:
done